pages/getting_started/index.tsx

```diff
@@ -609,109 +613,116 @@ export default function Onboarding(props: OnboardingProps) {
 export async function getServerSideProps(context: NextPageContext) {
   const session = await getSession(context);
 
-  let user: User = null;
   let integrations = [];
   let credentials = [];
   let eventTypes = [];
   let schedules = [];
+  if (!session?.user?.id) {
+    return {
+      redirect: {
+        permanent: false,
+        destination: "/auth/login",
+      } as const,
+    };
+  }
+  const user = await prisma.user.findFirst({
+    where: {
+      id: session.user.id,
+    },
+    select: {
+      id: true,
+      startTime: true,
+      endTime: true,
+      username: true,
+      name: true,
+      email: true,
+      bio: true,
+      avatar: true,
+      timeZone: true,
+      completedOnboarding: true,
+    },
+  });
+  if (!user) {
+    throw new Error(`Signed in as ${session.user.id} but cannot be found in db`);
+  }
 
-  if (session) {
```

Comment:
removed this massive block and replaced it with 

```ts
  if (!session?.user?.id) {
    return {
      redirect: {
        permanent: false,
        destination: "/auth/login",
      } as const,
    };
  }
```

Reply:
Smart fix on this.